CODE_REVIEW.md

```diff
@@ -0,0 +1,36 @@
+# code smells
```

Comment:
**General:**

- Watch typos and correct the phrases.
- Format files properly.

**Suggestion:** 

You can use any editor which will check for spelling and help correcting sentence and then replace the updated content here.

Also, provide line number wherever require. For example, Line no 9, you mentioned BEM naming convention to be followed but it's not clear for what. You can give some example.